glyphs2fontir/src/source.rs

```diff
@@ -996,8 +993,8 @@ mod tests {
             (72.0, 700.0),
         ] {
             let mut loc = UserLocation::new();
-            loc.set_pos("Optical Size", UserCoord::new(*opsz));
-            loc.set_pos("Weight", UserCoord::new(*wght));
+            loc.set_pos(Tag::from_str("opsz").unwrap(), UserCoord::new(*opsz));
```

Comment:
ditto these could just use`Tag::new`, which is infallible.

Reply:
Ty, will fix.